tests/ui/raw-ref-op/feature-raw-ref-op.rs

```diff
@@ -1,21 +0,0 @@
-// gate-test-raw_ref_op
```

Comment:
Remark: I almost thought this was a compiletest directive I missed while migrating. It might just be a copy-pasta from another test?

Reply:
No, `gate-test` exists in a bunch of places. It is interpreted by tidy, not compiletest, which is why it doesn't have the `@`.

Reply:
*oh* thanks, i didnt consider tidy